app/uk/gov/nationalarchives/omega/editorial/views/editSetRecordEdit.scala.html

```diff
@@ -159,21 +164,47 @@
 @govukSelect(Select(
     id = "legalStatus",
     name = "legalStatus",
-    items = legalStatusItems ++ legalStatusReferenceData.map {status =>
-            SelectItem(
-            selected=editSetRecordForm.data.get("legalStatus").contains(status.uri),
+    items = legalStatusItems ++ legalStatusReferenceData.map { status =>
+        SelectItem(
+            selected = editSetRecordForm.data.get("legalStatus").contains(status.uri),
             text = status.label,
             value = Some(status.uri)
-            )
-            },
+        )
+    },
     label = Label(forAttr = Some("legalStatus"), content = Text(messages("edit-set.record.edit.legal-status"))),
     errorMessage = editSetRecordForm.error("legalStatus").map { error =>
-            ErrorMessage(
+        ErrorMessage(
             content = Text(error.message),
             attributes = Map("for" -> "legalStatus")
+        )
+    },
+)
+)
+
+@govukSelect(Select(
+    id = "placeOfDeposit",
+    name = "placeOfDeposit",
+    items = {
+        val noSelectionAsMap = Map("" -> messages("edit-set.record.edit.place-of-deposit.no-selection"))
+        val corporateBodiesAsMap = noSelectionAsMap ++ corporateBodies.map(corporateBody => (corporateBody.id, corporateBody.name)).toMap
+        corporateBodiesAsMap
+                .map { case (id, label) =>
+            SelectItem(
+                selected = editSetRecordForm.data.get("placeOfDeposit").contains(id),
+                text = label,
+                value = Some(id).filter(_.nonEmpty),
+                disabled = id == ""
             )
+        }.toSeq
+    },
+    label = Label(forAttr = Some("placeOfDeposit"), content = Text(messages("edit-set.record.edit.place-of-deposit"))),
```

Comment:
@tna-erasmos should the error message for held by display the generic message and the default option in the dropdown show this message `edit-set.record.edit.place-of-deposit`

> You must choose an option

Reply:
I'll change it.

Reply:
Changed.

Reply:
LGTM